packages/xrpl/src/models/methods/tx.ts

```diff
@@ -41,35 +42,67 @@ export interface TxRequest extends BaseRequest {
   max_ledger?: number
 }
 
+/**
+ * Common properties of transaction responses.
+ *
+ * @category Responses
+ */
+interface BaseTxResult<
+  Version extends APIVersion,
+  T extends BaseTransaction = Transaction,
+> {
+  /** The SHA-512 hash of the transaction. */
+  hash: string
+  /**
+   * The Concise Transaction Identifier of the transaction (16-byte hex string)
+   */
+  ctid?: string
+  /** The ledger index of the ledger that includes this transaction. */
+  ledger_index?: number
+  /** Unique hashed string Transaction metadata blob, which describes the results of the transaction.
+   *  Can be undefined if a transaction has not been validated yet. */
```

Comment:
`meta_blob` field is absent if the user has not requested for the `binary` format too

Reply:
How would a user request for `binary` format? I'm having trouble finding it.